### DISCUSSION TITLE
Different across/down letters in a cell?

### DISCUSSION THREAD

**Discussion by DKMiller71** [viresh-ratnakar/exolve#104] [Q&A]
Some variety crosswords require the user to have a different letter in a cell going across vs. going down. Does Exolve support a way to do this? For example, in QXW, you can flag a cell that across/down need not agree, and gives you options to display.

**Comment by viresh-ratnakar** [comment]
Hmm, there's no way to directly do this, currently.

At first blush, I'm not inclined to add this feature (it seems to be contrary to the main quality of a "cross"word :-)).

Of course, we can create custom code for a puzzle to enable this.

I think you can also approximate it by using the "rebus cells" feature (https://github.com/viresh-ratnakar/exolve?tab=readme-ov-file#rebus-cells), which allows multiple letters to be entered in any cell. You may have to additionally use the `ignore-enum-mismatch` exolve-option).

Looks like turning on rebus-cells makes it OK to allow multiple letters to be entered in *any* cell. I should add a way to mark only specific cells where this is allowed. If you want to use this approach, then let me know, and I'll expedite this change.

**Reply by DKMiller71** [reply]
I was considering rebus cells as a backstop.  No, I didn't want to limit it to just certain cells (though interesting) because figuring out where the conflicts were is part of the overall puzzle.

The rebus approach does make a challenge for answer checking, since it would see both letters as part of both words across and down.

In QXW, you can specify cells don't have to agree, and it will actually track across vs. down.

And don't make an effort on my account, if you don't think anyone else might have interest.  I know what you mean and agree that it's not really in the spirit of a crossword.

Thanks for thinking about it!
